java/struts2/pom.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>org.apache.struts.benchmarker</groupId>
+    <artifactId>struts2</artifactId>
+    <version>1.0-SNAPSHOT</version>
+    <packaging>war</packaging>
+    <name>struts2</name>
+
+    <properties>
+        <struts2.version>2.5.22</struts2.version>
+        <log4j2.version>2.13.3</log4j2.version>
```

Comment:
is this required ? I mean to specify a version

Reply:
I can throw away the whole logging and just use what's provided.

Reply:
yeah, it will be awesome

The logging will be **disabled** for all frameworks